data/avro-schema/src/main/resources/avro/net/corda/data/membership/db/response/command/PersistApprovalRuleResponse.avsc

```diff
@@ -0,0 +1,13 @@
+{
+  "type": "record",
+  "name": "PersistApprovalRuleResponse",
+  "namespace": "net.corda.data.membership.db.response.command",
+  "doc": "Response to a persist approval rule request",
+  "fields": [
+    {
+      "name": "ruleId",
```

Comment:
It might be better to have the ID as part of the request (to allow idempotency).